urllib3/connectionpool.py

```diff
@@ -192,6 +194,8 @@ def __init__(self, host, port=None, strict=False,
         for _ in xrange(maxsize):
             self.pool.put(None)
 
+        self.transport_security_manager = transport_security_manager or TransportSecurityManager()
```

Comment:
I went ahead and removed the initialization, and changed the callers not to assume that `transport_security_manager` is set.